web/src/components/notification-pill/notification-pill.css

```diff
@@ -21,6 +20,18 @@
         background: var(--red);
     }
 
+    &.achievement {
+        background: var(--blue);
+        line-height: 1.5;
+        border-radius: 29px 0 0 29px;
```

Comment:
Since the "half-pill" slides in from the right, @mbransn probably had a different animation in mind than `fade-in`.

Maybe we should have a `.achievement` class to make the toast bigger and support an `img`, and an additional `.dashboard` class to make it a half-pill with a different animation? What do you think?

Reply:
Yeah, I will adjust the styles associated with the challenge dashboard.